layouts/partials/logo-section.html

```diff
@@ -17,7 +17,7 @@
       {{ $logo := printf "img/logos/%s" .logo | relURL }}
       <div class="column is-one-quarter">
         <a href="{{ .url }}" target="_blank">
-          <img src="{{ $logo }}">
+          	<img src="{{ $logo }}" alt=" Company icons">
```

Comment:
This partial is used for more than "Company icons".

```suggestion
          	<img src="{{ $logo }}" alt="Logo">
```

Reply:
Actually, we can probably do better than this by extracting the domain name from the `.url` and using that as `alt` text.